miro_preprocessor/Makefile

```diff
@@ -0,0 +1,53 @@
+ROOT = $(shell git rev-parse --show-toplevel)
+
+ifneq ($(ROOT), $(shell pwd))
+	include $(ROOT)/shared.Makefile
+endif
+
+
+xml_to_json_converter-build: xml_to_json_converter-test $(ROOT)/.docker/image_builder
+	$(ROOT)/scripts/run_docker_in_docker.sh \
+		-v $(ROOT):/repo \
+		image_builder:latest \
+		--project=xml_to_json_converter \
+		--file=/repo/miro_preprocessor/xml_to_json_converter.Dockerfile
```

Comment:
Minor: I don’t use the `/repo/` from this argument, because the working directory is `/repo/`, and that makes the command continue to work if we change the mount point in the container.

Not a massive issue tho.